### PR TITLE
#21146 - add import functionality to IndividualAssessment

### PR DESCRIPTION
**Pull request by daniwe4**
Trunk ticket 21146
add import functionality to IndividualAssessment

### CODE REVIEW

File: Modules/IndividualAssessment/classes/class.ilIndividualAssessmentDataSet.php [hunk 1]
```diff
@@ -105,4 +105,43 @@ protected function _readData($a_entity, $a_ids) {
...
+				}
+				else
+				{
+					$newObj = new ilObjIndividualAssessment();
+					$newObj->create();
+				}
+				$settings = new ilIndividualAssessmentSettings($newObj, $rec["content"], $rec["recordTemplate"]);
+				$newObj->setTitle($rec["title"]);
+				$newObj->setDescription($rec["description"]);
+				$newObj->setSettings($settings);
```
Comment (klees) [inline Modules/IndividualAssessment/classes/class.ilIndividualAssessmentDataSet.php]: It seems we are missing the "file required" and "place and time required" and "record template" here.

File: Modules/IndividualAssessment/classes/class.ilIndividualAssessmentImporter.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * Manual Assessment importer class
+ *
+ * @author Daniel Weise <daniel.weise@concepts-and-training.de>
+ */
+class ilIndividualAssessmentImporter extends ilXmlImporter
+{
+	/**
+	 * Init
+	 */
+	function init()
```
Comment (klees) [inline Modules/IndividualAssessment/classes/class.ilIndividualAssessmentImporter.php]: I can haz access qualifier here?

File: lang/ilias_de.lang [hunk 3]
```diff
@@ -13542,6 +13542,10 @@ iass#:#iass_notify#:#Teilnehmer nach abschließender Bewertung benachrichtigen
 iass#:#iass_finalize#:#Abschließen
 iass#:#il_iass_member#:#Teilnehmer
 iass#:#il_iass_members#:#Teilnehmer
+iass#:#iass_event_time_place_required#:#Zeit und Ort sind Pflichtangaben
+iass#:#iass_event_time_place_required_info#:#Zeit und Ort müssen mit angegeben werden.
+iass#:#iass_file_required#:#Datei ist eine Pflichtangabe
+iass#:#iass_file_required_info#:#Eine Datei muss beiligen.
```
Comment (klees) [inline lang/ilias_de.lang]: This does not relate to the original ticket, right?

**Review by klees — COMMENTED** [review]
LGTM in general, questions as always.

File: lang/ilias_de.lang (hunk 3 above)
Comment (daniwe4) [reply lang/ilias_de.lang]: Thats right. Should i remove it?

Comment (klees) [reply lang/ilias_de.lang]: Nope, boy- or girlscouting imo is always fine. Hints on that kind of stuff could go to the description of the PR to make it clear that this is in fact not about the bug.

Comment (matthiaskunkel) [reply lang/ilias_de.lang]: Just my 5 cent... It is "beiliegen" and not "beiligen". And all four new lang vars are real sentences (subject, verb, object) and should end with a dot.
(Sorry for just adding a comment and not offering a PR.)

Comment (daniwe4) [reply lang/ilias_de.lang]: Thanks. I will correct this immediately.

**Pull request merged**